llvm/lib/Target/AIE/AIE2PreLegalizerCombiner.cpp

```diff
@@ -83,6 +89,25 @@ AIE2PreLegalizerCombinerImpl::AIE2PreLegalizerCombinerImpl(
 {
 }
 
+bool AIE2PreLegalizerCombinerImpl::tryCombineAll(MachineInstr &MI) const {
+  if (tryCombineAllImpl(MI))
+    return true;
+
+  unsigned Opc = MI.getOpcode();
+  switch (Opc) {
+
+  case TargetOpcode::G_MEMCPY:
+  case TargetOpcode::G_MEMMOVE:
+  case TargetOpcode::G_MEMSET: {
+    if (InlineMemCalls &&
+        Helper.tryCombineMemCpyFamily(MI, 0 /*Use fed by TargetLowering*/))
```

Comment:
I guess this is needed for the second commit to work? Or is there any way to test this commit independently maybe?